module-2/backend/week_two.md

```diff
@@ -23,22 +37,35 @@ class Team << ActiveRecord::Base
   belongs_to :owner
 end
 ```
-
 Now how would you find the owner of the team with an id of 4?
+team.owner.name
 
 3. What do they allow you to do?
+Call on the singular of that column and have access to its attributes. 
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+Posting load code in a comment.
 8. Define foreign key, primary key, and schema.
+Schema - This holds the basic structure of your data-base. Migrations determine the makeup of the schema. Where tables live. 
+Primary Key - Unique identifier that every record is created with. 
+Foreign key - Link to another table through the unique identifier of a specific record in the second table. 
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
+They are the same? 
```

Comment:
Yup! I might add something to indicate that they define the relationship between the two tables.